test/e2e/storage/testsuites/base.go

```diff
@@ -37,8 +37,8 @@ import (
 type TestSuite interface {
 	// getTestSuiteInfo returns the TestSuiteInfo for this TestSuite
 	getTestSuiteInfo() TestSuiteInfo
-	// skipUnsupportedTest skips the test if this TestSuite is not suitable to be tested with the combination of TestPattern and TestDriver
-	skipUnsupportedTest(testpatterns.TestPattern, TestDriver)
+	// isTestSupported returns false if this TestSuite cannot be tested with the combination of TestPattern and TestDriver
```

Comment:
nit: "returns *true* if this TestSuite *can* be..."

Reply:
Done.